### PR TITLE
Wutao/update readme only

### PR DESCRIPTION
**Pull request by wutaomsft**
### Description
Updated "Getting Started Section" in README

### Related Issues
https://github.com/microsoft/recommenders/issues/1902

### Checklist:

- [X] I have followed the [contribution guidelines](../CONTRIBUTING.md) and code style for this project.
- [ ] I have added tests covering my contributions.
- [X] I have updated the documentation accordingly.
- [X] This PR is being made to `staging branch` and not to `main branch`.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -34,67 +34,56 @@ For a more detailed overview of the repository, please see the documents on the
...
-       ```
-
-    6. Run the [SAR Python CPU MovieLens](examples/00_quick_start/sar_movielens.ipynb)
-       notebook under the `00_quick_start` folder.  Make sure to
-       change the kernel to "Python (reco)".
-
-* For additional options to install the package (support for GPU,
-  Spark etc.) see [this guide](recommenders/README.md).
 
+<!--
```
Comment (miguelgfierro) [inline README.md:80]: What if we directly remove this?

File: README.md [hunk 2]
```diff
@@ -182,13 +171,14 @@ Smoke and integration tests are run daily on AzureML.
...
 ## Related projects
 
 - [Microsoft AI GitHub](https://github.com/microsoft/ai): Find other Best Practice projects, and Azure AI design patterns in our central repository.
 - [NLP best practices](https://github.com/microsoft/nlp-recipes): Best practices and examples on NLP.
 - [Computer vision best practices](https://github.com/microsoft/computervision-recipes): Best practices and examples on computer vision.
 - [Forecasting best practices](https://github.com/microsoft/forecasting): Best practices and examples on time series forecasting.
-
+-->
```
Comment (miguelgfierro) [inline README.md:181]: Same question here, what if we directly remove this?

File: README.md [hunk 3]
```diff
@@ -34,67 +34,56 @@ For a more detailed overview of the repository, please see the documents on the
...
 
 The installation of the recommenders package has been tested with 
-- Python versions 3.6 - 3.9 and [venv](https://docs.python.org/3/library/venv.html), [virtualenv](https://virtualenv.pypa.io/en/latest/index.html#) or [conda](https://docs.conda.io/projects/conda/en/latest/glossary.html?highlight=environment#conda-environment)
+- Python versions 3.7 - 3.9 and [venv](https://docs.python.org/3/library/venv.html), [virtualenv](https://virtualenv.pypa.io/en/latest/index.html#) or [conda](https://docs.conda.io/projects/conda/en/latest/glossary.html?highlight=environment#conda-environment)
 
 and currently does not support version 3.10 and above. It is recommended to install the package and its dependencies inside a clean environment (such as [conda](https://docs.conda.io/projects/conda/en/latest/glossary.html?highlight=environment#conda-environment), [venv](https://docs.python.org/3/library/venv.html) or [virtualenv](https://virtualenv.pypa.io/en/latest/index.html#)).
+-->
```
Comment (miguelgfierro) [inline README.md:77]: Kind of same question as before, what do you think of removing this as well. In the new getting started, there is enough information

File: README.md [hunk 4]
```diff
@@ -34,67 +34,56 @@ For a more detailed overview of the repository, please see the documents on the
...
+<!--
+In addition to the core package, several extras are provided, including:
++ `[examples]`: Needed for running examples.
++ `[gpu]`: Needed for running GPU models.
++ `[spark]`: Needed for running Spark models.
++ `[dev]`: Needed for development for the repo.
++ `[all]`: `[examples]`|`[gpu]`|`[spark]`|`[dev]`
++ `[experimental]`: Models that are not throughly tested and/or may require additional steps in installation).
++ `[nni]`: Needed for running models integrated with [NNI](https://nni.readthedocs.io/en/stable/).
+-->
```
Comment (miguelgfierro) [inline README.md:69]: This part is very interesting. Thinking as a user, in a quick view, I could understand that there are multiple flavours. I think if we add this we skip the user one step (that I would make) which is to install recommenders[examples], run one of the GPU examples, get an error, then go back, realize that I had to do recommenders[gpu,examples], and then run it. What do you think?

**Review by miguelgfierro — APPROVED** [review]
Looks great

**Review by anargyri — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (wutaomsft) [reply README.md:80]: yes will do.

File: README.md (hunk 3 above)
Comment (wutaomsft) [reply README.md:77]: yes will do.

**Pull request merged**